release/src/main/scripts/publish_docker_images.sh

```diff
@@ -62,6 +62,11 @@ if [[ $confirmation = "y" ]]; then
   docker push apachebeam/go_sdk:latest
 
   echo '-------------Generating and Pushing Flink job server images-------------'
+  FLINK_VER=("$(ls -1 runners/flink | awk '/^[0-9]+\.[0-9]+$/{print}')")
```

Comment:
Missing `docker rmi` for Flink images. (the other file has it)

Reply:
The published images are meant to be permanent.

Reply:
Look below at lines 81 onwards (I can't comment there because github).

Reply:
It's deleting the local images only.